README.md

```diff
@@ -49,3 +49,18 @@ To only translate the CQL and not do the rest of the build/test steps, spin up a
 docker run -d -p 8080:8080 cqframework/cql-translation-service
 yarn translate
 ```
+
+### Utility Function Assurance Testing
+
+The `assurance` folder contains unit tests and relevant fixtures used for testing the functionality of the helper functions used in the testing harness. 
+
+Example CQL for assurance testing lives in the `assurance/fixtures/cql` subdirectory. To build this CQL into ELM, spin up a cql-translation-service docker container and run the `yarn:translate` script:
+``` bash
+docker run -d -p 8080:8080 cqframework/cql-translation-service
+yarn translate:assurance
+```
```

Comment:
We might want to provide instructions for tearing down the `cql-translation-service` when people are done, since this runs the container in a detached manner. Thoughts @mgramigna